docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -161,30 +145,34 @@ In a dual-region setup, a loss of a region will invariably affect Camunda 8, reg
 
-The [operational procedure](./../../operational-guides/multi-region/dual-region-ops.md) looks in detail at short-term recovery from a region loss and how to long-term fully re-establish the lost region. The procedure works the same way for active or passive region loss since we don't consider traffic routing (DNS) in the scenario.
+The [operational procedure](./../../operational-guides/multi-region/dual-region-ops.md) looks in detail at a recovery from a region loss and how to long-term fully re-establish the lost region.
+
+::caution
+Customers are expected to proactively monitor for regional failures and take ownership of executing the necessary [operational procedures](./../../operational-guides/multi-region/dual-region-ops.md) to ensure smooth recovery and failover.
+::
 
 ### Active region loss
 
-The loss of the active region results in the following:
+The total loss of the active region results in the following:
 
-- **Loss of Data**: Data previously available in Operate and Tasklist is no longer accessible.
 - **Service Disruption**: Traffic routed to the active region can no longer be served.
-- **Workflow Engine Failure**: The workflow engine stops processing due to quorum loss.
+- **Workflow Engine Impact**: The workflow engine stops processing due to quorum loss.
+- **Loss of Tasklist and Operate operations**: Uncompleted batch operations and Tasklist assignments will be lost
 
 #### Steps to take in case of active region loss
 
 1. **Temporary Recovery:** Follow the [operational procedure for temporary recovery](./../../operational-guides/multi-region/dual-region-ops.md#failover) to restore functionality and unblock the workflow engine.
 2. **Traffic Rerouting:** Reroute traffic to the passive region, which will now become the new active region.
 3. **Data and Task Management:** Due to the loss of data in Operate and Tasklist:
-   1. Reassign any uncompleted tasks in Tasklist.
+   1. Reassign any uncompleted tasks in the Tasklist.
    2. Recreate batch operations in Operate.
-4. **Permanent Region Setup:** Follow the [operational procedure to create a new permanent region](./../../operational-guides/multi-region/dual-region-ops.md#failback) that will become your new passive region.
+4. **Permanent Region Setup:** In case of permanent/complete region loss, follow the [operational procedure to create a new permanent region](./../../operational-guides/multi-region/dual-region-ops.md#failback) that will become your new passive region.
 
 ### Passive region loss
 
 The loss of the passive region results in the following:
 
-- **Workflow Engine Impact**: The workflow engine will stop processing due to the loss of quorum.
+- **Workflow Engine Impact**: The workflow engine will stop processing tasks due to the loss of quorum.
```

Comment:
Mhm, tasks sounds like a subset of things in Zeebe.
Like user task or service task, could be misunderstood.

It stops processing processes sounds stupid.
Maybe we could reword it?